applications/asset_tracker/src/main.c

```diff
@@ -441,14 +442,23 @@ static void modem_rsrp_data_send(struct k_work *work)
 /**@brief Poll device info and send data to the cloud. */
 static void device_status_send(struct k_work *work)
 {
-	int len;
 	char data_buffer[MODEM_INFO_JSON_STRING_SIZE] = {0};
+	int len;
+	int ret;
 
 	if (!atomic_get(&send_data_enable)) {
 		return;
 	}
 
-	len = modem_info_json_string_get(data_buffer);
+	ret = modem_info_params_get(&modem_param);
+
+	if (ret < 0) {
+		printk("ret: %d\n", ret);
```

Comment:
Could we produce meaningful messages?
Any person using this library will waste some time in order to find what this message means and what is the cause of the encountered problem.

Reply:
Yes, of course. I've changed the wording and added some additional LOG_DBG print options.